source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
Let's do the copy functions etc. in another pull request when/if they are needed (right now they aren't). The point with this PR  was basically to move `_copyFloatPoint3DToFloatMatrixColumn` in #425 from `Quaternion` to `FloatMatrix` and to add a `getColumn` function.